src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./Filter.css"
+
+interface IFilter {
+    selectedRegion: string,
+    onChange(event: any): any
```

Comment:
`onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void`

Reply:
This is what I'd like to know, I didn't found what type use for the event, thanks!